tfsdk/serve_test.go

```diff
@@ -1719,6 +1719,7 @@ func TestServerPlanResourceChange(t *testing.T) {
 				"favorite_colors": tftypes.NewValue(tftypes.List{ElementType: tftypes.String}, []tftypes.Value{
 					tftypes.NewValue(tftypes.String, "red"),
 					tftypes.NewValue(tftypes.String, "orange"),
+					tftypes.NewValue(tftypes.String, "yellow"),
```

Comment:
This is a test fix since the configuration contained it, so the proposedNewState would also contain it.